prowler/providers/github/services/repository/repository_service.py

```diff
@@ -127,11 +128,32 @@ def _list_repositories(self):
                                 repo.security_and_analysis.secret_scanning.status
                                 == "enabled"
                             )
+                        try:
+                            # Use get_dependabot_alerts to check if Dependabot alerts are enabled
+                            repo.get_dependabot_alerts()[0]
+                            # If the call succeeds, Dependabot is enabled (even if no alerts)
+                            dependabot_alerts_enabled = True
+                        except Exception as error:
+                            error_str = str(error)
+                            if (
+                                "403" in error_str
+                                and "Dependabot alerts are disabled for this repository."
+                                in error_str
+                            ):
+                                dependabot_alerts_enabled = False
+                            elif "403" in error_str or "404" in error_str:
+                                dependabot_alerts_enabled = None
+                            else:
```

Comment:
```suggestion
                            else:
```
I would log all the rest of errors too.